Tests/Unit/EmogrifierTest.php

```diff
@@ -1519,4 +1542,45 @@ public function emogrifyAutomaticallyClosesUnclosedTag()
             $this->subject->emogrify()
         );
     }
+
+    /**
+     * @test
+     */
+    public function emogrifyReturnsCompleteHtmlDocument()
+    {
+        $this->subject->setHtml($this->html5DocumentType . '<html><body><p></p></body></html>');
+
+        self::assertSame(
+            $this->html5DocumentType . self::LF .
+            '<html>' . self::LF .
+            '<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>' . self::LF .
+            '<body><p></p></body>' . self::LF .
+            '</html>' . self::LF,
+            $this->subject->emogrify()
+        );
+    }
+    
+    /**
+     * @test
+     */
+    public function emogrifyReturnsBodyContentFromHtml()
```

Comment:
Sorry for the to and fro.

Actually, the test names usually should start with the name of the tested method (i.e., emogrifyBodyContent) and also be a complete sentence. So the tests should be named emogrifyBodyContentReturnsBodyContentFromHtml and Content.